ssl/ssl_lib.c

```diff
@@ -7211,3 +7260,220 @@ int SSL_stream_conclude(SSL *ssl, uint64_t flags)
     return 0;
 #endif
 }
+
+int SSL_add_expected_rpk(SSL *s, EVP_PKEY *rpk, int nid)
```

Comment:
```suggestion
int SSL_add_expected_rpk(SSL *s, EVP_PKEY *rpk, int mtype)
```
It isn't a `nid` anymore.